solutions/arrays/sum-query.js

```diff
@@ -0,0 +1,43 @@
+let fs = require("fs");
+let data = fs.readFileSync(0, 'utf-8');
+let idx = 0;
+data = data.split('\n');
+
+function readLine() {
+  idx++;
+  return data[idx - 1];
+}
+
+// -------- Do NOT edit anything above this line ----------
+// Use readLine() for taking input, it will read one line of from the input  and is stored in string format
+
+let firstArray=readLine().split(" ")
```

Comment:
give spaces around binary operators, please check the coding conventions and also add ";" at the end of each line